bitbots_vision_tools/scripts/extract_from_rosbag.py

```diff
@@ -0,0 +1,115 @@
+#!/usr/bin/env python
+
+import rosbag
+import argparse
+import os
+from sensor_msgs.msg import Image
+import cv_bridge
+import cv2
+from cv_bridge import CvBridge
+import numpy as np
+
+try:
+    input = raw_input
+except NameError:
+    pass
+
+parser = argparse.ArgumentParser("Extract images from a rosbag")
+
+parser.add_argument("-o", "--out-dir", required=True, help="Output directory for the images", dest="outputdir")
+parser.add_argument("-i", "--input-bag", required=True, help="Input rosbag", dest="inputfile")
+parser.add_argument("-n", "--nth-image", help="Every n-th image will be saved, prompted if not specified", dest="n",
```

Comment:
I think n should default to 1 because extracting every image is the most common scenario. The prompt could then be omitted.

Reply:
I have discussed about this with @jgueldenstein and I agree with his point, that it is really not necessary to always extract every image and prompting the question about this parameter increases the awareness and makes the user think twice...